index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+   <div class="Container">
+     <div class="Header">
+       <h3>Welcome</h3>
+       <img src="images/Enabled.png" alt="">
+     </div>
+   
+
+   <div class="headline">
```

Comment:
- [ ] The container of this section should be a `section `tag, this is meant to increase the accessibility of your page, it makes your tags more meaningful and this is a rule of the [semantic HTML](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#semantics_in_html)